index.js

```diff
@@ -14,17 +20,25 @@ module.exports = (css, settings) => {
   const optionData = settings.sassOptions && settings.sassOptions.data || "";
   const data = optionData + "\n" + cssWithPlaceholders;
 
+  const file = settings.babel && settings.babel.filename;
```

Comment:
I am not sure about this, I thought with v1 we agreed that this should work when setting the `includePath` via `sassOptions`.  cc @atombender

Reply:
Relative paths do not work properly in `v1`, they can easily resolve to the wrong file if you have two files with the same name in different directories

```yaml
# directory structure
components
  foo-component
    styles.scss
    foo-component.js    # @import './styles.js'
  bar-component
    styles.scss
    bar-component.js    # @import './styles.js' 
```

```js
sassOptions.includePaths = [
  "components/foo-component",
  "components/bar-component"
]
```

In this scenario, with your current implementation, it is going to search in order of the includePaths. This means both imports will resolve to the file inside `foo-component`. That's definitely not what `bar-component` wanted.

I am happy to add a regression tests that proves all that :arrow_up: if it increases your comfort level, but we're really just testing `node-sass` implementation details at that point. The test as I've got it checks that explicit relative imports will work without modifying `includePaths`. 

The problem you resolved in `v1` was that you were clobbering the users intended  `includePaths` settings in an effort to get relative paths to work.

Reply:
Gotcha thank you for the exhaustive explanation. How do you solve this issue in a standalone node-sass setup?

Reply:
The problem with the current implementation is that `node-sass` doesn't know the file location, it is only being provided the files contents. In a standalone `node-sass` setup, that would never happen (it's reading in the file contents itself, and it already knows the file path from the CLI or config).

This is simply giving node-sass the extra bit of information it needs to process the `@import` command properly.